### PR TITLE
Added links for keyword files in chapter 4.3

### PR DESCRIPTION
**Pull request by hakonhagland**
Used the script in #410 to add links to all keyword files in Chapter 4.3. See #410 for more information.

### CODE REVIEW

File: parts/chapters/subsections/4.3/SKIP300.fodt [hunk 1]
```diff
@@ -4205,12 +4204,12 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
...
...e commercial <text:span text:style-name="T20">compositional</text:span> simulator until the <text:a xlink:href="#__RefHeading___Toc605472_3199477706">ENDSKIP</text:a> keyword is encountered. All keywords between the <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a><text:span text:style-name="T20">3</text:span>00 and <text:a xlink:href="#__RefHeading___Toc605472_3199477706">ENDSKIP</text:a> keywords are ignored by the commercial <text:span text:style-name="T20">compositional</text:span> simulator. The <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a><text:span text:style-name="T20">3</text:span>00 keyword is ignored by the commercial <text:span text:style-name="T20">black-oil</text:span> simulator. Each <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a><text:span text:style-name="T20">3</text:span>00 keyword should be paired with an <text:a xlink:href="#__RefHeading___Toc605472_3199477706">ENDSKIP</text:a> keyword. See also the <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a> and <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a><text:span text:style-name="T20">1</text:span>00 keywords. <text:s/></text:p>
```
Comment (blattms) [inline parts/chapters/subsections/4.3/SKIP300.fodt:4207]: I think here the initial formatting `SKIP<text:span ..>3</text:span>00` broke the link. The some links  should go to SKIP300 and not SKIP

File: parts/chapters/subsections/4.3/SKIP300.fodt [hunk 2]
```diff
@@ -4205,12 +4204,12 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
...
+    <text:p text:style-name="P60">If the following example is read using the commercial <text:span text:style-name="T20">compositional</text:span> simulator then the <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a><text:span text:style-name="T20">3</text:span>00 keyword will cause the black-oil PVT data to be ignored, <text:span text:style-name="T20">the input will continue to be read after the next <text:a xlink:href="#__RefHeading___Toc605472_3199477706">ENDSKIP</text:a> keyword is encountered</text:span>, <text:span text:style-name="T20">and</text:span> the <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a><text:span text:style-name="T20">1</text:span>00 keyword will be ignored and the compositional PVT data will be read <text:span text:style-name="T20">along with</text:span> the remainder of the <text:span text:style-name="T20">input</text:span>.</text:p>
```
Comment (blattms) [inline parts/chapters/subsections/4.3/SKIP300.fodt:4212]: I think here the initial formatting `SKIP<text:span ..>3</text:span>00` broke the link. The some links  should go to SKIP300 and not SKIP

**Review by blattms — COMMENTED** [review]
Some issues  with links that should actually go to SKIP300. We should probably fix the formatting and reapply the script.

File: parts/chapters/subsections/4.3/SKIP300.fodt (hunk 1 above)
Comment (hakonhagland) [reply parts/chapters/subsections/4.3/SKIP300.fodt]: Yes, good catch! I have updated the PR. I removed all span tags I could find related to SKIP100 and SKIP300 in `SKIP300.fodt` and then reapplied the script. Can you check again if I missed some?

**Pull request merged**